packages/ceramic-common/src/utils/doctype-utils.ts

```diff
@@ -211,4 +227,20 @@ export class DoctypeUtils {
             throw new Error(`Validation Error: ${errorMessages}`)
         }
     }
+
+    /**
+     * Checks if record is signed DTO ({jws: {}, linkedBlock: {}})
+     * @param record - Record
+     */
+    static isSignedRecordDTO(record: any): boolean {
+        return typeof record === 'object' && ('jws' in record && 'linkedBlock' in record)
+    }
+
+    /**
+     * Checks if record is signed
+     * @param record - Record
+     */
+    static isSignedRecord(record: any): boolean {
+        return typeof record === 'object' && record.link && record.payload && record.signatures
```

Comment:
Seems this function will not explicitly return a boolean, might be better use `in` or an explicit cast to boolean

Reply:
yep 👍